### ISSUE TITLE
qustions abount mse loss 

### ISSUE THREAD

**Issue by phexic** [tju-maoyan/AMNet#1]
self.loss = (1.0 / args.batch_size) * tf.nn.l2_loss(self.Y_ - self.Y)  # MSE loss
should be changed to the following code？
self.loss = (1.0 / args.batch_size) * tf.losses.mean_squared_error(self.Y_, self.Y)
